internal/output/const.go

```diff
@@ -0,0 +1,15 @@
+package output
+
+const (
+	TypeStringDefault = "default"
+	TypeStringPretty  = "pretty"
+	TypeStringCSV     = "csv"
+	TypeStringJSON    = "json"
+)
+
+const (
+	ValueUnknown    = "UNKNOWN"
+	ValueNotDecoded = "*"
+	ValueSkip       = ">"
+	ValueNil        = "-"
```

Comment:
IMO, instead of showing `-` to user, `nil`or `null` makes more sense.

Reply:
We can later change this if necessary.